grid.v3/client.go

```diff
@@ -300,16 +299,18 @@ func (c *Client) getWireClient(ctx context.Context, nsReceiver string) (WireClie
 	return cc.client, nil
 }
 
```

Comment:
:) I originally called it Minus from the SQL operator from the same name.   difference also fine.